pkg/processor/runtime/python/wrapper.py

```diff
@@ -234,15 +234,23 @@ def get_next_packet(sock, buf):
     return packet
 
 
+def should_encode_body(response):
+    if is_py2:
+        return False
+
+    return isinstance(response['body'], bytes)
+
+
 def response_from_handler_output(handler_output):
     """Given a handler output's type, generates a response towards the
     processor"""
 
     response = {
         'body': '',
-        'content_type': 'text/plain',
+        'content_type': 'text-plain',
```

Comment:
Won't this be returned to the user? We need to return valid content types. If this is indeed the case and this would've triggered a bug, please add a test.

Reply:
Oops, my bad.